implementations/rust/ockam/ockam_transport_websocket/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ockam_transport_websocket"
-version = "0.12.0"
```

Comment:
We usually bump every crate's version automatically during release process. Could you please remove it? Otherwise it gets bumped twice

Reply:
Ok, got it!